Datastore/tests/Unit/DatastoreClientTest.php

```diff
@@ -69,7 +69,8 @@ public function testGrpcConnection()
         $this->checkAndSkipGrpcTests();
 
         $client = TestHelpers::stub(DatastoreClient::class, [[
-            'transport' => 'grpc'
+            'projectId' => self::PROJECT,
```

Comment:
This change might not be needed. We are only initialising and asserting on the class.

Reply:
(same as below) I added it because the tests were failing without it. With our mono-repo setup, the Project ID environment variable gets set in an unrelated test that propagates through the rest of the tests. When the tests are run individually, an error is thrown for a missing projectID.